svthalia/concrexit
# Add alternate_logo to Partner

Closes #1650



### Summary
An optional, alternate logo can be uploaded to partners. If the alternate logo is not present, then the main logo will be shown on the main page.
### How to test
Steps to test the changes you made:
1. Go to [admin website](/partners/partner/)
2. Click on any partner
3. Upload an alternate logo
4. It should now be shown in the home bage.
